main/envs.py

```diff
@@ -186,9 +185,10 @@ def parse_list(name, value, default):  # pylint: disable=unused-argument
             parsed_value = [
                 item.strip(" ").lstrip('"').rstrip('"')
                 for item in parsed_value.lstrip("[").rstrip("]").split(",")
+                if item
```

Comment:
Previously, if we add a `list` in the `.env` file `e.g: VALUE_LIST=[first_value,]` it will be rendered as `['first_value', '']`. After applying this fix, that anomaly will be gone.